tests/reports/test_study_utils.py

```diff
@@ -232,6 +232,7 @@ def distinct_strings_with_common_characters(draw):
     distinct_strings_with_common_characters(), st.integers(min_value=1, max_value=10)
 )
 @settings(**hypothesis_settings)
+@pytest.mark.hypothesis
```

Comment:
We've had to register custom marks in other changes I've seen, is this a mark that hypothesis creates for us or have I just misunderstood why the other changes needed to register their marks?

Reply:
I forgot about registering it. Is that just for documentation then? Happy to register it for that reason though.

Reply:
I'm assuming you're talking about adding an entry into pyproject.toml anyway. Our other markers haven't been registered with pytest and I assume they're being picked up automatically.

Reply:
I saw it in databuilder and used it in job-server on the [faulty] assumption that it was required.

Definitely looks like it's [just for documentation](https://docs.pytest.org/en/7.1.x/example/markers.html#registering-markers).  I can see the use but don't feel strongly either way for this change, so I'll leave it up to you :)